app/controllers/characters_controller.rb

```diff
@@ -69,6 +69,19 @@ def update
     end
   end
 
+  def duplicate
+    @dup = @character.dup
```

Comment:
Should I then rescue it and display a nice message? (If so, what's the error type?)